src/r_main.c

```diff
@@ -967,32 +967,32 @@ void R_InitAnyRes(void)
 void R_BindRenderVariables(void)
 {
   BIND_NUM_GENERAL(extra_level_brightness, 0, 0, 4, "Level brightness");
-  BIND_BOOL_GENERAL(stretchsky, false, "1 to stretch short skies");
-  BIND_BOOL_GENERAL(linearsky, false, "1 for linear horizontal sky scrolling");
-  BIND_BOOL_GENERAL(r_swirl, false, "1 to enable swirling animated flats");
-  BIND_BOOL_GENERAL(smoothlight, false, "1 to enable smooth diminishing lighting");
+  BIND_BOOL_GENERAL(stretchsky, false, "Stretch short skies");
+  BIND_BOOL_GENERAL(linearsky, false, "Linear horizontal scrolling for skies");
+  BIND_BOOL_GENERAL(r_swirl, false, "Swirling animated flats");
+  BIND_BOOL_GENERAL(smoothlight, false, "Smooth diminishing lighting");
   M_BindBool("voxels_rendering", &default_voxels_rendering, &voxels_rendering,
-             true, ss_none, wad_no, "1 to enable voxels rendering");
+             true, ss_none, wad_no, "Allow rendering of voxels");
```

Comment:
`Allow voxel model rendering`

Reply:
`Allow rendering of voxel models`?

Reply:
That's good too.

Reply:
"voxel models"

Reply:
I insist on having "Allow" at the start, because the user needs to provide them.

Reply:
Right, I think the critical part was "voxels" versus "voxel models". Let's go with the latter.